src/openloco/game_commands.h

```diff
@@ -242,6 +243,22 @@ namespace openloco::game_commands
     }
 
     // Build company headquarters
+    // Note: The game seems to call do_55, then do_54 in success case and returns 0x0 code
+    //       if we try to build over the existing only do_54 is called and it returns 0x80000000 (in regs.ebx)
```

Comment:
54 will be remove headquarters. You need to remove the ghost before placing the real one.

Reply:
Argh sorry I might have this back to front. I can't quite work out what way round things are ATM. Maybe add a commented stating that this is probably Remove company headquarters.